spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'spec_helper'
+
+RSpec.describe 'User integrations', type: :feature do
+  it 'displays user profile in show page' do
+    user = create(:user)
+    visit user_path(user)
+    expect(page).to have_content('Andrew Carnegie')
+  end
+
+  it 'Can see user name ' do
+    user = create(:user)
+    visit user_path(user)
+    expect(page).to have_content('Andrew Carnegie')
+  end
```

Comment:
- [ ] [REQUIRED] While it's good that the test checks for the presence of content related to the user's name ('Andrew Carnegie'), it does not specifically check for the user's actual username.

Here is what your test should look like:

``` ruby
it 'displays user username in show page' do
  user = create(:user, username: 'acarnegie')
  visit user_path(user)
  expect(page).to have_content(user.username)
end
```